monkey/infection_monkey/network/info.py

```diff
@@ -21,6 +22,41 @@
 RTF_REJECT = 0x0200
 
 
+class IntRange:
+    """
+    Represents an inclusive range of integers, with a step size of 1.
```

Comment:
Whether or not the range is inclusive is up to the user, I think. There's no data that's part of this class that would enforce any inclusivity constraints.